src/views/EdgeApplicationsRulesEngine/FormFields/FormFieldsEdgeApplicationsRulesEngine.vue

```diff
@@ -65,34 +69,40 @@
     { label: 'does not exist', value: 'does_not_exist' }
   ])
 
+  const showLabelApplicationAccelerator = computed(() => {
+    if (props.showRequiresApplicationAcceleratorInDescription) return ''
+
+    return ' - requires Application Accelerator'
```

Comment:
Se não me engano era aqui que a Carol comentou para colocar em upper
```suggestion
    return ' - Requires Application Accelerator'
```